source/index.js

```diff
@@ -197,9 +270,22 @@ class Storyblok {
           return reject(res)
         }
 
+        if (typeof params.resolve_relations !== 'undefined' && params.resolve_relations.length > 0) {
+          await this.resolveRelations(response.data, params)
+        }
+
         if (params.version === 'published' && url != '/cdn/spaces/me') {
           provider.set(cacheKey, response)
         }
+
+        if (response.data.cv) {
+          cacheVersions[params.token] = response.data.cv
+
+          if (params.version == 'draft' && cacheVersions[params.token] != response.data.cv) {
```

Comment:
second half of this conditional will always be `false` cuz you're setting it to that value the line before.